Source/Manager/IdentityAPI/Router.swift

```diff
@@ -23,6 +23,7 @@ enum Router {
     case requiredFields(userID: String)
     case client(clientID: String)
     case product(userID: String, productID: String)
+    case devices(userID: String)
```

Comment:
This no longer takes the `userID` as a route parameter (instead basing the user id on the included token).